### ISSUE TITLE
Pausieren, stoppen, ausblenden

### ISSUE THREAD

**Issue by lukasniebler** [FAU-Brand/Lastenheft#75]
## Rechtsquelle
EN 9.2.2.2 (Web)
WCAG 2.2.2 (A)

## Anforderungsbeschreibung
Für sich bewegende, blinkende, scrollende oder sich automatisch aktualisierende Informationen gelten alle folgenden Punkte:

### Sich bewegend, blinkend, scrollend:

Für alle sich bewegenden, blinkenden oder scrollenden Informationen, die (1) automatisch beginnen, (2) länger als 5 Sekunden dauern und (3) parallel zu anderen Inhalten dargestellt werden, gibt es einen Mechanismus für den Benutzer, um diese zu pausieren, zu beenden oder auszublenden, außer die Bewegung, das Blinken oder das Scrollen ist Teil einer Handlung, bei der es unentbehrlich ist und

### Automatische Aktualisierung:

Für alle sich automatisch aktualisierenden Informationen, die (1) automatisch beginnen und (2) parallel mit anderen Inhalten dargestellt werden, gibt es einen Mechanismus, damit der Benutzer die Aktualisierung pausieren, beenden oder ausblenden oder die Häufigkeit der Aktualisierung kontrollieren kann, außer die automatische Aktualisierung ist Teil einer Handlung, bei der sie unentbehrlich ist.

ANMERKUNG 1: Für Anforderungen in Bezug auf flackernden oder blitzenden Inhalt beachten Sie
[Richtlinie 2.3 "Anfälle und physische Reaktionen" der WCAG 2.1 (Englisch)](https://www.w3.org/TR/WCAG21/#seizures-and-physical-reactions)
.

ANMERKUNG 2: Jeglicher Inhalt, der dieses Erfolgskriterium nicht erfüllt, kann die Möglichkeit eines Benutzers beeinträchtigen, die ganze Website/das gesamte Dokumente/die ganze Software zu nutzen. Daher muss jeglicher Inhalt auf einer Webseite/in einem Dokument/in einer Software (egal ob er dazu benutzt wird, andere Erfolgskriterien zu erfüllen oder nicht) dieses Erfolgskriterium erfüllen. Siehe
[Konformitätsbedingung 5.2.5 "Nicht störend" der WCAG 2.1 (Englisch)](https://www.w3.org/TR/WCAG21/#cc5)
.

ANMERKUNG 3: Inhalt, der regelmäßig durch Software aktualisiert oder zum Benutzeragenten gestreamt wird, muss Informationen, die zwischen dem Beginn der Pause und der Wiederaufnahme der Präsentation generiert oder erhalten werden, nicht aufrechterhalten oder anzeigen, da dies möglicherweise technisch nicht möglich ist und es in vielen Situationen irreführend sein könnte, dies zu tun.

ANMERKUNG 4: Eine Animation, die als Teil einer Vorlade-Phase oder einer ähnlichen Situation entsteht, kann als unentbehrlich angesehen werden, wenn für alle Benutzer während dieser Phase keine Interaktion stattfinden kann und wenn es Benutzer verwirren könnte oder dazu führen könnte, dass sie denken, dass der Inhalt eingefroren oder defekt ist, wenn der Fortschritt nicht angezeigt wird.
